modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepository.java

```diff
@@ -353,33 +360,18 @@ protected Experiment internalGetExperiment(Application.Name appName,
      */
     @Override
     public Experiment.ID createExperiment(NewExperiment newExperiment) {
-
-        validator.validateNewExperiment(newExperiment);
-
-        // Ensure no other experiment is using the label
```

Comment:
@bargenilesh Major/Question: Why was this check removed? Are we checking for it elsewhere in the new code? Am I missing something, here?